### ISSUE TITLE
Can you please describe how to import/export auto completion data

### ISSUE THREAD

**Issue by gokceng** [zsh-users/zsh-autosuggestions#710]
### Is your feature request related to a problem? Please describe.
My laptop had to be wiped and even though I backed up `.zshrc` & `.zsh_history` files, I couldn't figure out how I can make `zsh-autosuggestions` recognise the history. I've checked some commands in `.zsh_history` are not suggested when I use terminal.

### Describe the solution you'd like
If you can describe how to backup / import auto suggestion data, that'd be great.

### Describe alternatives you've considered
I'd be fine with any alternative as long as I can make `zsh-autosuggestions` process entries in `.zsh_history`

**Comment by ivan** [comment]
zsh-autosuggestions uses the zsh history. There's no extra data.

**Comment by gmahe** [comment]
Got HISTFILE set to `~/.zsh_custom_history` and if I run the command `history` I get all those imported data. Though I still won't get suggestion of the one I have re-run on this machine. 
Any reason?

**Comment by dekadentno** [comment]
I also didn't manage to restore autosuggestions on my other VM. Can anyone please write ELI5 steps to achieve this?

I'm open to develop a tool/script that will automate this if I find out how to backup/restore it.
